zipper/protocols/v2/protobuf_group.go

```diff
@@ -360,10 +361,10 @@ func (c *ClientProtoV2Group) doTagQuery(ctx context.Context, isTagName bool, que
 	logger := c.logger
 	var rewrite *url.URL
 	if isTagName {
-		logger = logger.With(zap.String("type", "tagName"))
+		logger = logger.With(zap.String("type", "tagName"), zap.String("context_uuid", utilctx.GetUUID(ctx)))
```

Comment:
I would name that field not a `context_uuid` but rather `carbonapi_uuid` to be consistent with all other places in carbonapi.